Config.uk

```diff
@@ -19,4 +19,16 @@ if LIBMIMALLOC
 	config MIMALLOC_ASSERT
 		bool "Enable assertions"
 		default n
+	config LIBMIMALLOC_TEST
+		bool "Enable libmimalloc tests"
+		select LIBUKTEST
+		select LIBMIMALLOC_TEST_API
+		default n
+		help
+			Enables all libmimalloc tests.
+	if LIBMIMALLOC_TEST
+		config LIBMIMALLOC_TEST_API
+			bool "Enable libmimalloc api test suite"
+			default n
```

Comment:
```suggestion
			default y
```
Just like `uktest` selects `test all libraries`, the default should be once one selects the tests for one library, all the tests for that library are selected by default and can be unselected if needed.

Reply:
Fixed, its much better now, before the `Enable libmimalloc api test suite` was not modifiable. now we can uncheck it, if we want to select a specific set of test suites.